apps/new-dealer/acceptance/pages/obtain.js

```diff
@@ -1,5 +1,86 @@
 'use strict';
 
+const TRANSLATIONS = require('../../translations/en/default');
+const FIELDS = TRANSLATIONS.fields;
+const PAGES = TRANSLATIONS.pages;
```

Comment:
I like this pattern too. We've been doing it on the other forms so just been consistent here